shared/output/cloud/speech/lib/google/cloud/speech/v1/speech/client.rb

```diff
@@ -80,26 +80,17 @@ class OperationsClient < Google::Longrunning::OperationsClient
             # @param scopes [Array<String>]
             #   The OAuth scopes for this service. This parameter is ignored if an
             #   updater_proc is supplied.
-            # @param client_config [Hash]
-            #   A Hash for call options for each method. See Google::Gax#construct_settings
-            #   for the structure of this data. Falls back to the default config if not
-            #   specified or the specified config is missing data points.
             # @param timeout [Numeric]
             #   The default timeout, in seconds, for calls made through this client.
             # @param metadata [Hash]
             #   Default metadata to be sent with each request. This can be overridden on a
             #   per call basis.
-            # @param exception_transformer [Proc]
-            #   An optional proc that intercepts any exceptions raised during an API call to
-            #   inject custom error handling.
             #
             def initialize \
                 credentials: nil,
                 scopes: ALL_SCOPES,
-                client_config: {},
                 timeout: DEFAULT_TIMEOUT,
                 metadata: nil,
-                exception_transformer: nil,
```

Comment:
I'd like to either keep this in scope for this PR and figure out a better way to do it now, or open an issue so we don't forget to add it back. We do need to support this feature.

Reply:
Let's track in #107.